client/components/tinymce/plugins/contact-form/dialog/field.jsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
+import omit from 'lodash/object/omit';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import FieldHeader from './field-header';
+import FoldableCard from 'components/foldable-card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+import FormCheckbox from 'components/forms/form-checkbox';
+import SelectDropdown from 'components/select-dropdown';
+import DropdownItem from 'components/select-dropdown/item';
+import TokenField from 'components/token-field';
+import getLabel from './locales';
+
+export default React.createClass( {
+	displayName: 'ContactFormDialogField',
+
+	mixins: [ PureRenderMixin ],
+
+	propTypes: {
+		label: PropTypes.string.isRequired,
+		type: PropTypes.string.isRequired,
+		options: PropTypes.string,
+		required: PropTypes.bool,
+		onUpdate: PropTypes.func.isRequired,
+		onRemove: PropTypes.func.isRequired
+	},
+
+	renderOptions() {
+		if ( this.props.type === 'radio' || this.props.type === 'select' ) {
+			let { options } = this.props;
+			if ( options && typeof options === 'string' ) {
+				options = options.split( ',' );
+			}
+
+			return (
+				<FormFieldset>
+					<FormLabel>{ this.translate( 'Options' ) }</FormLabel>
+					<TokenField
+						value={ options || [] }
+						onChange={ tokens => this.props.onUpdate( { options: tokens.join() } ) }/>
+				</FormFieldset>
+			);
+		}
+
+		return null;
+	},
+
+	render() {
+		const fieldTypes = [ 'name', 'email', 'checkbox', 'select', 'radio', 'text', 'textarea', 'url' ];
+		const remove = <Gridicon icon="trash" onClick={ this.props.onRemove } className="editor-contact-form-modal-field__remove" />;
+
+		return (
+			<FoldableCard
+				header={ <FieldHeader { ...omit( this.props, [ 'onRemove', 'onUpdate' ] ) } /> }
+				icon={ 'pencil' }
```

Comment:
A consistency point, since there's mixed usage here, but this can be `icon="pencil"`